doc/releases/releases.yml

```diff
@@ -12,6 +12,11 @@
 # If a version might represent an actual number (e.g. 0.80) quote it.
 #
 releases:
+  reef:
+    target_eol: 2026-08-01
```

Comment:
I think this should be 2025 (i.e. two years from now)?

Reply:
Good catch. Thanks, @idryomov.